_posts/IMBLAH/2021-11-10-IMBLAH-project-idea.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 author: IMBLAH
-title: “Bo's Project idea & work plan”
+title: Bo's Project idea & work plan
```

Comment:
Thanks for removing these curly quotes `“`; other students have had unanticipated problems using them in their projects and posts. However, it is safest to enclose this title string in straight quotes `"Bo's Project idea & work plan"`.